Assets/css/styles.css

```diff
@@ -0,0 +1,373 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+body{
+  font-family: "Roboto-condensed", sans-serif;
+  width: 100%;
+  background-color: #f4f5f6;
+}
+.main-container{
+  width: 85%;
+  background-color: #fff;
+  margin: auto;
+}
+img{
+  width: 100%;
+}
+p{
+  color: black;
+}
+.page-header{
+  background-color: #f72210;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+}
+.row p{
+  font-weight: 900;
+  color: #fff;
+}
+.nav{
+  width: 100%;
+  background-color: #fff;
+  color: black;
+  font-weight: bold;
+}
+.nav a{
+  text-decoration: none;
+  color: black;
+}
+.nav-pills{
+  border-bottom: 1px solid #aaa;
+}
+.row i{
+  color: #aaa;
+  padding-top: 10px;
+}
+.row input{
+  border: 0px;
+}
+.text-right.header-item {
+  padding: 40px 20px 0px 20px;
+}
+.text-right.header-item button{
+  color: red;
+  font-weight: bold;
+  border: 0px;
+  background-color: #fff;
+}
+.text-right.header-item i{
+  color: red;
+}
+.text-left.header-item{
+  padding: 40px 20px 0px 20px;
+}
+.news h3{
+  color: black;
+}
+.news a{
+  text-decoration-color: #f72210;
+}
+.col-sm-5.col-md-5.mid-col a{
+  text-decoration-color: #f72210;
+}
+.col-sm-5.col-md-5.mid-col h3{
+  color: black;
+}
+.container-fluid{
+  margin-top: 15px;
+  height: 100%;
+}
+.container-fluid p{
+  color: black;
+}
+#rocks{
+  width: 100%;
+}
+body > div > div:nth-child(4) > div.row > div.col-sm-3.col-md-3.left-col > div:nth-child(1) > button{
+  font-size: 12px;
+  width:100%;
+}
+.text-muted{
+  font-size: 18px;
+  font-weight: bold;
+}
+.news h3{
+  margin-top: 10px;
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 24px;
+  color: #000;
+}
+.news a{
+  text-decoration-color: #f72210;
+}
+.news p{
+  font-weight: 400;
+}
+ul.right-bar-ul{
+  padding-left: 0px;
+}
+.right-bar-ul li{
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+.right-bar-ul img{
+  width: 60px;
+  height: 60px;
+}
+.right-bar-ul a{
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
+.info a:hover{
+  color: #f72210;
+}
+.info{
+  padding: 0px 5px 0px 15px;
+  line-height: 1.4em;
+}
+.aut{
+  display: flex;
+  flex-direction: row;
+}
+.aut p{
+  font-weight: 400;
+  color: #555;
+}
+.aut a{
+  color: #555;
+}
+#abs{
+  width: 100%;
+}
+.align-top p{
+  font-size: 16px;
+}
+.align-top a:hover{
+  text-decoration-color: #f72210;
+}
+.par-mid{
+  font-weight: 400;
+}
+.col-sm-4.col-md-4.mid-col a{
+  color: black;
+  font-size: 24px;
+}
+.col-sm-4.col-md-4.mid-col a:hover{
+  text-decoration-color: #f72210;
+}
+.red-links p:hover{
+  color: #f72210;
```

Comment:
Okay, do you see how many times it repeats, while I get it may get long, it may be best to group them and it has gone past being just a suggestion but a high recommendation!

like 
`.class1,
  .class2,
  .class3,
{
   color: #f72210;
}
`